psydac/linalg/iterative_solvers.py

```diff
@@ -256,6 +258,10 @@ def jacobi(A, b):
     from psydac.linalg.block   import BlockMatrix, BlockVector
     from psydac.linalg.stencil import StencilMatrix, StencilVector
 
+    # In case A is None we return a zero vector
+    if A is None:
+        return b.copy()*0.
```

Comment:
Slightly more efficient solution, as it avoids the multiplication and a temporary:
```suggestion
    # In case A is None we return a zero vector
    if A is None:
        return b.space.zeros()
```
This should also work for a block vector.

Reply:
done in [abd53a7](https://github.com/pyccel/psydac/pull/213/commits/abd53a7daa92a80586489809d8c7b9c4578cd34b)